chromadb/api/local.py

```diff
@@ -177,14 +194,14 @@ def _modify(
         self,
         id: UUID,
         new_name: Optional[str] = None,
-        new_metadata: Optional[Dict] = None,
-    ):
+        new_metadata: Optional[Metadata] = None,
```

Comment:
This type is wrong, metadata is for _embedding_ metadata, not collection metadata. Please use CollectionMetadata.